src/Dhall/Main.hs

```diff
@@ -65,6 +68,13 @@ parseOptions = Options <$> parseMode <*> parseExplain <*> parsePlain
             <>  Options.Applicative.help "Disable syntax highlighting"
             )
 
+    parseInplace =
```

Comment:
Yeah, I think this should be a `format`-specific option (See https://github.com/dhall-lang/dhall-haskell/pull/451 for an example of how to do that)

Note that it's still worth keeping the intermediate `parseInplace` utility as I imagine that other utilities might reuse it in the future (like the `dhall freeze` command described in https://github.com/dhall-lang/dhall-haskell/issues/437)

Reply:
:ok_hand: